### PR TITLE
[parallel-hashmap] Update version to v1.3.11

### PR DESCRIPTION
**Pull request by greg7mdp**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [x] SHA512s are updated for each updated download
- [x] The "supports" clause reflects platforms that may be fixed by this new version
- [x] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: ports/parallel-hashmap/portfile.cmake [hunk 1]
```diff
@@ -3,8 +3,8 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO greg7mdp/parallel-hashmap
-    REF v1.3.8
-    SHA512 499f132aa1513942762ae5214e9a37915529d60142439d32276d9e68a446c93ecea66caa32ac00bea683e2c932101cd999c8839a89211e91032f777f236181f7
+    REF v1.3.11
```
Comment (Cheney-W) [inline ports/parallel-hashmap/portfile.cmake:6]:
```suggestion
    REF "v${VERSION}"
```

File: versions/p-/parallel-hashmap.json [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
+    {
+      "git-tree": "4236193fdd2e7cd1959cd7dbafbdf7c74691617f",
+      "version": "1.3.10",
+      "port-version": 0
+    },
+    {
+      "git-tree": "964973167eefcd25c8654b0da68cbfda9043232c",
+      "version": "1.3.9",
+      "port-version": 0
+    },
```
Comment (Cheney-W) [inline versions/p-/parallel-hashmap.json:17]:
```suggestion
```
Only one version is added to each modified port's versions file.

**Comment by Cheney-W** [comment]
If you have executed `./vcpkg x-add-version --all` once before, then after modifying the port file again, you need to run `./vcpkg x-add-version --all --overwrite-version` instead of `./vcpkg x-add-version --all`.

**Comment by BillyONeal** [comment]
Thanks for the update!

**Pull request merged**